### DISCUSSION TITLE
RAUC & labgrid community meetup at Embedded World 2024 in Nürnberg

### DISCUSSION THREAD

**Discussion by jluebbe** [labgrid-project/labgrid#1337] [Announcements]
📅 Date: Wednesday, April 10th
🕒 Time: 14:30 - 15:30
📍 Location: Cafe in Hall 4, located at 4-640a

Join us for a casual gathering to talk about these open-source projects. It's a good opportunity to meet fellow developers and users in the labgrid & RAUC communities, exchange ideas, and discuss future directions.

No need to RSVP—just come by if you're interested. See you at #ew24!

**Comment by aparcar** [comment]
@KarlK90 ping

**Reply by AndreasWilligWES** [reply]
Thanks for the Meetup, as a reminder, a majordomo for labgrid would be nice (similar to barebox) to get at least those announcements

**Reply by jluebbe** [reply]
We don't want to add another communication channel just for announcements. You can enable notifications for GitHub discussions on the [main page](https://github.com/labgrid-project/labgrid). There is a [request for more restrictive notifications](https://github.com/orgs/community/discussions/3951) open.

Also, we post important messages on Mastodon (https://fosstodon.org/@labgrid) and the chat channel (bridgd IRC&Matrix).

**Reply by AndreasWilligWES** [reply]
Okay, i did subscribe, but still i would think it is a good idea to also share questions, ideas and best-practices (solutions) in a channel by side of GitHub

**Reply by sjg20** [reply]
Can someone please invite me to fosstodon.org ?

**Reply by jluebbe** [reply]
> Can someone please invite me to fosstodon.org ?

You should have received an invite via mail. If anyone else needs one, you can ask me at jlu@pengutronix.de.
